components/LoginForm.vue

```diff
@@ -84,25 +84,26 @@ export default Vue.extend({
         this.$auth.setUser(response.data.User);
         this.$user.set(response.data.User.Id, this.serverUrl, accessToken);
       } catch (error) {
-        let errorMessage = 'Unexpected Error';
+        let errorMessage = this.$t('unexpectedError');
 
         if (!error.response) {
-          errorMessage = 'Server Not Found';
+          errorMessage = this.$t('serverNotFound');
         } else if (error.response.status === 500) {
-          errorMessage = 'Incorrect Password';
+          errorMessage = this.$t('incorrectPassword');
```

Comment:
this could actually be incorrect username/password

Reply:
Hm, then the message definitely needs changing

Reply:
rename to `Incorrect Username or Password`